### DISCUSSION TITLE
TET & ACT keep non routable sections intact

### DISCUSSION THREAD

**Discussion by 0709wiwiwi** [devemux86/cruiser#533] [Show and tell]
Convert a good quality gpx (record) track into a gpx navtrack inc TBT.
Upload in the [RouteYou web planner](https://www.routeyou.com/route/add) and download as Locus gpx (only premium users)

**Comment by WolfgangZ42** [comment]
That's true! RouteYou creates a perfect navigable route in Locus format that can be imported by Cruiser. But at the cost of many, many nonsensical shaping points (more than 1600 in this case). I tested it with one day of TET Portugal (because TET routes are license-free, unlike ACT): 
<img width="200" src="https://github.com/devemux86/cruiser/assets/121806238/a6fb96b9-3b65-4db1-911c-438b814271a2"/> - - - -     <img width="200" src="https://github.com/devemux86/cruiser/assets/121806238/28ac18c1-b7f5-4628-9bcb-38abb2513eb6"/>
I could even reverse the route in Cruiser for my usage - in RouteYou only possible with subscription. 
Import into Cruiser doesn't work with other exports from RouteYou:
<img width="200" src="https://github.com/devemux86/cruiser/assets/121806238/a8c459ea-6058-41ee-874c-6fa65fe6f073"/>
The corresponding files: [TET_P-5.zip](https://github.com/devemux86/cruiser/files/15360192/TET_P-5.zip)
